compiler/src/dotty/tools/dotc/cc/Setup.scala

```diff
@@ -555,17 +556,18 @@ class Setup extends PreRecheck, SymTransformer, SetupAPI:
           sym == defn.AnyClass
```

Comment:
I think this clause is now redundant. `AnyClass` is not a pure class.

Reply:
Thanks! it is now fixed